mintapi/signIn.py

```diff
@@ -38,28 +42,50 @@
 INPUT_CSS_SELECTORS_LABEL = "input_css_selectors"
 SPAN_CSS_SELECTORS_LABEL = "span_css_selectors"
 BUTTON_CSS_SELECTORS_LABEL = "button_css_selectors"
+MFA_PROMT_TEXT = "mfa_promt_text"
```

Comment:
This is misspelt.  It should actually be "prompt".

Reply:
Nice catch thanks!